README.md

```diff
@@ -0,0 +1,72 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#  Catalog of my things
+
+> In this project we have practiced Object Oriented Programming by adding classes and using OOP principles, Describe the SOLID principles of OOP, Insert and query data in SQL, Create a UML class diagram, Store data in files, Build interactive console apps, and Recognize the value of making equal contributions to group projects to produce the best outcome.
+
+
+| Starting | Options|
+|-----------------|---------------|
+| ![Starting](./img1.jpg) | ![Options](./img2.jpg) |
+
+
+## Built With
+
+- Ruby
+- OOP
+- SQL Postgres
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+- Click on the top right green "code" button.
+- On the dropdown menu, choose "download with zip" button.
+- After download, extract the zip file and you have the project on your machine.
+- Run `rubu main.rb` to run the application.
```

Comment:
Please add the `bundle install` command in README so the user can install project dependencies.
Please correct the typo here as it should be `ruby main.rb`.
Please add instructions for the user to run the tests.